app/controllers/posts_controller.rb

```diff
@@ -3,7 +3,6 @@ class PostsController < ApplicationController
 
```

Comment:
- [ ] You have done an excellent job, the roles and abilities are set up correctly! 💯  However, it would be great if you **load and authorize the resource** in your posts controller, this would authorize every action. You can see more [here](https://github.com/ryanb/cancan/wiki/authorizing-controller-actions)

Reply:
I just made the change